### DISCUSSION TITLE
How to edit file in boot partition?

### DISCUSSION THREAD

**Discussion by rrlevy** [tesla-android/issue-tracker#95] [Q&A] [answered]
The install guide tells you to overclock the pi by editing a file in the boot partition.

How are we supposed to do that if we can't access any files on the SD card when plugging it on Windows computer? (it asks to format the drive)

Is there a way to ssh into Android and do that?

**Comment by RetoHB** [comment]
Having the same issue and question...

**Comment by mikegapinski** [comment, marked as answer]
I don’t know how it’s done on windows but on Mac and Linux the boot partition appears as a removable drive when you insert the sdcard into your computer.

It’s a FAT partition so it should appear on Windows also.

The overclocking process on Tesla Android is the same as on every other raspberry pi operating system, there is a ton of resources online explaining how to get it done on Windows

**Comment by RetoHB** [comment]
All online resources I found ssh into the PI. Can we ssh into Tesla Android?
The SD card cannot be mounted on Windows. Also installing a Linux file system mount tool did not help...

**Comment by rrlevy** [comment]
I ended up finding a Mac and using it to edit the boot partition, and it worked

**Comment by RetoHB** [comment]
What type is the boot partition? I have no access to a Mac so need to find a solution on Windows.

**Comment by RetoHB** [comment]
Windows 7 seems to work as well. Only 10+ has a problem with it...

**Comment by Bbuzby5656** [comment]
I boot up the machine using a build of raspian - insert the Android card in a card reader - and edit the boot - config.txt from there
